database/mysql.go

```diff
@@ -187,15 +186,21 @@ func CreateTestMySQLDB(t *testing.T) *TestMySQLDB {
 
 // We connect as root to MySQL server and create database with random name to
 // run our migrations on it later.
-func createTemporaryDatabase(config *MySQLConfig) (string, error) {
+func createTemporaryDatabase(t *testing.T, config *MySQLConfig) (string, error) {
 	dsn := fmt.Sprintf("%s:%s@%s/", "root", config.Password, config.Address)
 	db, err := sql.Open("mysql", dsn)
 	if err != nil {
 		return "", err
 	}
 	defer db.Close()
 
-	dbName := "test" + base.ID()
+	// Set dbName to something like "TestCreateTemporaryD.._Jun_24_20:24:46"
+	dbName := fmt.Sprintf(
+		"%s.._%s",
+		t.Name()[:20],
+		time.Now().Local().Format(time.Stamp),
```

Comment:
Are spaces allowed in database names? Why not a format like t.Name() + "YYYY-MM-DD-HHmmss" ?

Reply:
Right, but the format doesn't work.

```
Error 1064: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '.._Jun_25_15:02:27' at line 1
```